doc/reference/key_extraction.html

```diff
@@ -2099,7 +2099,7 @@ <h3><a name="ckey_result_semantics">Semantics of
 <blockquote>
 <code>tuple&lt;Q0,...,Qk></code>, <code>k &lt;= n</code><br>
 <code>composite_key_result&lt;composite_key&lt;K0,...,Kk> ></code>, with
-<code>Ki::result_type = Qi</code> for all <code>i = 0,...,k</code>.
+<code>Ki::result_type = Qi</code> for all <code>i = 0,...,k</code>
```

Comment:
Again, I'd replace the period with a comma. Also, the clause

<code>tuple&lt;Q0,...,Qk></code>, <code>k &lt;= n</code>

is missing a final comma.